packages/bioformats/superbuild.cmake

```diff
@@ -7,8 +7,8 @@ set(bf-git-url "" CACHE STRING "URL of Bio-Formats git repository")
 set(bf-git-branch "" CACHE STRING "URL of Bio-Formats git repository")
 
 # Current stable release.
-set(RELEASE_URL "http://downloads.openmicroscopy.org/bio-formats/5.1.4-rc1/artifacts/bioformats-5.1.4-rc1.zip")
-set(RELEASE_HASH "SHA512=d88aff72aba03393f1de1bc57fbf9d94431cee177a4944ebc32de1dc864c64f724d86571b973f8a555133e4ca4d88b3798e1bcba0d25f95f696b3d7fa837561e")
+set(RELEASE_URL "http://downloads.openmicroscopy.org/bio-formats/5.1.4/artifacts/bioformats-dfsg-5.1.4.zip")
```

Comment:
We can use either here.  This makes it match the homebrew build, and we don't need any of the non-GPL content from the non-dfsg version.